xbmc/interfaces/python/PythonSwig.cpp.template

```diff
@@ -117,6 +126,8 @@ void doMethod(Node method, MethodType methodType)
     else
       return;
   }
+  else if (method.@name == "contains")
```

Comment:
Okay, I get it. If the class has a 'contains' method then you're going to be able to call it using sequence ops from python. Will this break classes that simply have a method called 'contains' or will it just provide an alternative way of calling it from python?